sql/updates/characters/3.3.5/2020_09_02_00_characters.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `character_battleground_data` 
+MODIFY COLUMN `mountSpell` int(10) UNSIGNED NOT NULL DEFAULT 0 AFTER `taxiEnd`;
```

Comment:
From SQL checks of Circle CI:
>Checking database 'characters' for missing filenames in tables...
"sql/updates/characters/2020_09_02_00_characters.sql" is missing in the 'characters'.'updates' table.

These changes need to be applied to https://github.com/TrinityCore/TrinityCore/blob/3.3.5/sql/base/characters_database.sql and the update needs to be added there too